### PR TITLE
Update issues in binary commander docker and source pages in core docs 2.1.5 - closes #595

### LINKED ISSUE #595: Update issues in binary, commander, docker & source pages in Core docs 2.1.5
Setup>Binary/Source: All networks listed, but it’s not Testnet/Mainnet compatible
Setup>Commander: Note only 2.0 or higher. We are on 3.0 docs.
Setup>Docker/Source: Order of OS should always be the same. And Linux/Ubuntu should be decided upon.

Parent issue #591

### PR DESCRIPTION
**Pull request by chris529**
closes #595

### CODE REVIEW

**Review by Tschakki — CHANGES_REQUESTED** [review]
As discussed, please keep Testnet and Mainnet in this version (and remove betanet and alphanet if present).

**Comment by chris529** [comment]
Yes, you are right Mona, it is now done.

File: modules/ROOT/pages/setup/binary.adoc [hunk 1]
```diff
@@ -83,6 +94,10 @@ To connect your node to the Mainnet, execute the following command:
 bash installLisk.sh install -r main
 ----
 --
+====
+
+[tabs]
+====
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/binary.adoc]: Remove these lines

File: modules/ROOT/pages/setup/binary.adoc [hunk 2]
```diff
@@ -127,8 +143,8 @@ For further information and how to administer your Lisk node, please see the xre
 
 If you are not running Lisk locally, you will need to follow the xref:{url_config_api_access}[Configuration - API] document to enable access.
 
-With all of the above steps complete you are ready to move on to the configuration documentation if you wish to enable forging or SSL, please see the xref:{url_config}[General configuration].
+If all of the above steps have been successfully completed, then the next step is to move on to the configuration documentation. If you wish to enable forging or SSL, please see the xref:{url_config}[General configuration].
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/binary.adoc]: Please use a new line for each sentence

File: modules/ROOT/pages/setup/binary.adoc [hunk 3]
```diff
@@ -127,8 +143,8 @@ For further information and how to administer your Lisk node, please see the xre
...
 
 == Post-installation (optional)
 
-* Recommended: Set up a xref:{url_config_logrotation}[log rotation]
+* In this case it is recommended to set up a xref:{url_config_logrotation}[log rotation].
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/binary.adoc]:
```suggestion
* It is recommended to set up a xref:{url_config_logrotation}[log rotation].
```

File: modules/ROOT/pages/setup/commander.adoc [hunk 4]
```diff
@@ -43,60 +30,50 @@ If you have already performed these, please proceed to the <<install, Installati
...
-Don't run this command with a user with `sudo` rights. +
-If you do so, the installation will be aborted, as it is not necessary and not recommended to install Lisk Core for a user with `sudo` rights.
+Do not run this command with a user that has `sudo` rights. +
+If this is executed then the installation will be aborted, as it is not required and not recommended to install Lisk Core for a user with `sudo` rights.
 ====
 
 [TIP]
 ====
-You can freely choose the name of your Lisk Core instance.
+Please choose the name of your Lisk Core instance.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc]: Imho, we shouldn't use 'please' too often in the docs. Also it is changing the sence of the sentence here a little imho. The tip just says it is possible to change the name, not that the user should do it.

File: modules/ROOT/pages/setup/commander.adoc [hunk 5]
```diff
@@ -43,60 +30,50 @@ If you have already performed these, please proceed to the <<install, Installati
...
 
-Just be aware that it is necessary to refer to your custom name instead of `lisk-mainnet` or `lisk-testnet` when running the commands.
+Please note that it is necessary to refer to your custom name instead of `lisk-mainnet` or `lisk-testnet` when running the commands.
 ====
 
 [tabs]
 ====
-Mainnet::
+Betanet::
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc]: This tabs are not correct, please just restore the original content...

File: modules/ROOT/pages/setup/commander.adoc [hunk 6]
```diff
@@ -170,8 +147,8 @@ lisk core:status lisk-testnet
 
 Check you network settings to verify the corresponding ports are open.
 
-It is also recommended to set up a xref:{url_config_logrotation}[log rotation].
+In addition it is also recommended to set up a xref:{url_config_logrotation}[log rotation].
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc:161]: Is this `in addition` really needed here?

File: modules/ROOT/pages/setup/docker.adoc [hunk 7]
```diff
@@ -14,8 +14,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_docker_install_windows: https://docs.docker.com/docker-for-windows/install/
 :url_docker_linux_post_install: https://docs.docker.com/install/linux/linux-postinstall/
 :url_lisk_core_repository: https://github.com/LiskHQ/lisk-core
-:url_lisk_explorer_testnet: https://testnet-explorer.lisk.io/
-:url_lisk_explorer_mainnet: https://explorer.lisk.io/
+:url_lisk_explorer_betanet: https://betanet-explorer.lisk.io/
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/docker.adoc:19]: Again... Mainnet and Testnet should stay here

File: modules/ROOT/pages/setup/docker.adoc [hunk 8]
```diff
@@ -36,43 +35,82 @@ Please refer to {url_docker_install}[https://docs.docker.com/engine/installation
...
-Install `make` using your package manager.
-For example, use `apt-get` if running Ubuntu:
+include::partial$create-a-new-user.adoc[]
+
+
+=== Create a new user
+
+To run and manage a Lisk Core node in the future, please create a separate `lisk` user as described below:
+
+// [tabs]
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/docker.adoc]: Tabs format is broken here

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/setup/binary.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/setup/binary.adoc]: OK, now done.. :-)

File: modules/ROOT/pages/setup/commander.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/setup/commander.adoc]: Yes, I see your point Mona, OK it is now resolved.. :-)

File: modules/ROOT/pages/setup/commander.adoc (hunk 5 above)
Comment (chris529) [reply modules/ROOT/pages/setup/commander.adoc]: OK, this should now work..

File: modules/ROOT/pages/setup/commander.adoc (hunk 6 above)
Comment (chris529) [reply modules/ROOT/pages/setup/commander.adoc:161]: Well, we could live without it, however I think it fits well within the flow of this paragraph. Nevertheless, we can always change it if you wish Mona.. For example,  we could have something like this..."At this stage it is recommended to set up a log rotation" ?   :-)

File: modules/ROOT/pages/setup/docker.adoc (hunk 7 above)
Comment (chris529) [reply modules/ROOT/pages/setup/docker.adoc:19]: OK, they should be fixed now :-)

File: modules/ROOT/pages/setup/commander.adoc [hunk 9]
```diff
@@ -80,6 +67,10 @@ To verify your node is running correctly, execute the following command:
 lisk core:status lisk-mainnet
 ----
 --
+====
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/commander.adoc]: The tabs format is broken again... please just restore the original content, no need to change anything here.

File: modules/ROOT/pages/setup/commander.adoc (hunk 6 above)
Comment (Tschakki) [reply modules/ROOT/pages/setup/commander.adoc:161]: Ok thanks for explanation. I think its cool like that 👍

File: modules/ROOT/pages/setup/docker.adoc [hunk 10]
```diff
@@ -36,36 +37,43 @@ Please refer to {url_docker_install}[https://docs.docker.com/engine/installation
...
 
 Install `make` using your package manager.
 For example, use `apt-get` if running Ubuntu:
 
 [source,bash]
+
 ----
 sudo apt-get install curl make
 ----
+====
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/docker.adoc]: Tabs format broken, please fix.

File: modules/ROOT/pages/setup/docker.adoc [hunk 11]
```diff
@@ -74,6 +82,47 @@ These are the default ports for connecting with the network, they can be altered
 
 include::partial$create-a-new-user.adoc[]
 
+
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/docker.adoc]: Why is this content added here?

File: modules/ROOT/pages/setup/docker.adoc [hunk 12]
```diff
@@ -207,7 +256,7 @@ The result should appear as shown below:
 
 When your node is synchronised, the values of `networkHeight` and `height` should be (nearly) equal.
 
-To fully verify that your node is synchronised with the network, go to the {url_lisk_explorer_mainnet}[Lisk Explorer(Mainnet)] or {url_lisk_explorer_testnet}[Lisk Explorer(Testnet)] and compare the network height in the explorer with the height of your node.
+To fully verify that your node is synchronised with the network, go to the {url_lisk_explorer_betanet}[Lisk Explorer(Betanet)] and compare the network height in the explorer with the height of your node.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/docker.adoc]: This should not be changed...

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/setup/commander.adoc (hunk 9 above)
Comment (chris529) [reply modules/ROOT/pages/setup/commander.adoc]: OK, I think this is OK now, could you please check it Mona, thx . :-)

**Review by Tschakki — APPROVED** [review]
Looking good now, thanks!

**Pull request merged**